src/components/Card/Card.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import './assets'
+interface ReusableCardProps {
```

Comment:
Interfaces are used for validating the structure of an object, it is not used a storage variable.